<?xml version="1.0"?>
<launch>

  <!-- start tof camera viewer -->
  <node pkg="cob_camera_sensors" type="all_camera_viewer" name="all_camera_viewer" cwd="node" respawn="false" output="screen">
	<remap from="right/image_color" to="stereo/right/image_raw"/>
	<remap from="left/image_color" to="stereo/left/image_raw"/>
	<remap from="right/camera_info" to="stereo/right/camera_info"/>
	<remap from="left/camera_info" to="stereo/left/camera_info"/>
	<remap from="image_xyz" to="tof/image_xyz"/>
	<remap from="image_grey" to="tof/image_grey"/>
	<remap from="save_camera_images" to="cob_camera_sensors/save_camera_images"/>

	<param name="absolute_output_directory_path" value="$(find cob_camera_sensors)/common/files/cob3-1/calib/" />

	<param name="use_right_color_camera" value="true" />
	<param name="use_left_color_camera" value="true" />
	<param name="use_tof_camera" value="true" />
  </node>

</launch>
